### DISCUSSION TITLE
Comp Guidelines Document

### DISCUSSION THREAD

**Discussion by honzikschenk** [team401/2024-Robot-Code#107] [General]
I started a [document](https://docs.google.com/document/d/1TCCeuDcqBEqSYdKysgANXI7t4N5pY07NMdYNIil80Kw/edit?usp=drivesdk) for highlighting programming "best practices"/guidelines during comp. I thought it'd be useful to keep the entire process streamlined.

Do y'all have any suggestions on what else to add?

**Comment by PChild** [comment]
Thoughts on [commit on deploy](https://github.com/FRC2713/Robot2024/blob/5ca8d768db9a4c288540350b5609d05113120196/build.gradle#L199) instead of sticky notes for changes?

**Reply by jkleiber** [reply]
This is preferred by me ^ we just need to ensure it doesn't get set to run on any non-competition branches

**Reply by honzikschenk** [reply]
Just created a ticket for this here: #122
